resources/views/livewire/order/purchase.blade.php

```diff
@@ -9,6 +9,37 @@
         </x-modal>
     </div>
 @endsection
+@section('state-card')
+    <x-card>
+        <div class="space-y-3">
+            <x-select
+                :label="__('Authorisation user')"
+                wire:model="order.approval_user_id"
```

Comment:
Doesn't this mean, that you can change the authorization user to yourself and than "approve/confirm" the order?